bazel/README.md

```diff
@@ -97,34 +106,49 @@ for how to update or override dependencies.
     version of `ar` on the PATH, so if you run into issues building third party code like luajit
     consider uninstalling binutils.
 
-    On Windows, additional dependencies are required:
-
-    Install the [MSYS2 shell](https://msys2.github.io/) and install the `diffutils`, `patch`,
-    `unzip`, and `zip` packages using `pacman`. Set the `BAZEL_SH` environment variable to the path
-    of the installed MSYS2 `bash.exe` executable. Setting the `MSYS2_ARG_CONV_EXCL` environment
-    variable to a value of `*` is often advisable to ensure argument parsing in the MSYS2 shell
-    behaves as expected.
-
-    `Git` is required. The version installable via MSYS2 is sufficient.
+    ### Windows
+    On Windows, you'll need to install several dependencies manually.
 
-    Install the Windows-native [python3](https://www.python.org/downloads/), the POSIX flavor
+    [python3](https://www.python.org/downloads/): Specifically, the Windows-native flavor. The POSIX flavor
     available via MSYS2 will not work. You need to add a symlink for `python3.exe` pointing to
     the installed `python.exe` for Bazel rules which follow POSIX conventions. Be sure to add
     `pip.exe` to the PATH and install the `wheel` package.
+    ```
+    mklink %USERPROFILE%\Python38\python3.exe %USERPROFILE%\Python38\python.exe
+    set PATH=%PATH%;%USERPROFILE%\Python38
+    set PATH=%PATH%;%USERPROFILE%\Python38\Scripts
+    pip install wheel
+    ```
     
-    For building with MSVC (the `msvc-cl` config option), you must install at least the VC++
-    workload from the
-    [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019). 
+    [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019):
+    For building with MSVC (the `msvc-cl` config option), you must install at least the VC++ workload.
     You may also download Visual Studio 2019 and use the Build Tools packaged with that
     installation. Earlier versions of VC++ Build Tools/Visual Studio are not recommended at this
     time. If installed in a non-standard filesystem location, be sure to set the `BAZEL_VC`
     environment variable to the path of the VC++ package to allow Bazel to find your installation
     of VC++. Use caution to ensure the `link.exe` that resolves on your PATH is from VC++ Build Tools and
     not MSYS2.
+    ```
+    set BAZEL_VC=%USERPROFILE%\VSBT2019\VC
+    set PATH=%PATH%;%USERPROFILE%\VSBT2019\VC\Tools\MSVC\14.26.28801\bin\Hostx64\x64
```

Comment:
nit: the path command might be a bit brittle if the version changes.

Reply:
All of these commands are brittle to varying degrees. My main motivation was to provide an example of how the setup can be done. The details for each reader can certainly change. Maybe a disclaimer at the top for saying "the provided Windows commands are examples, ensure correct paths"?

Reply:
I agree, I think it is really valuable and it helped me a lot when I was looking into the commands.